mirar/processors/sources/namer.py

```diff
@@ -91,28 +99,40 @@ def get_next_name(self, detection_time: Time, last_name: str = None) -> str:
         :return: new name
         """
         cand_year = detection_time.datetime.year % 1000
+
+        if last_name is not None:
```

Comment:
Is this necessary?

Reply:
I put it in for testing, but figured it was needed on NYE. I can remove it though if you prefer. It's not the end of the world if the NYE candidates have 23 names.

Reply:
I think it will not affect WINTER ever, but we should leave it in for other surveys